profiles/coreos/base/package.use.mask

```diff
@@ -1,5 +1,5 @@
 # We don't ship GnuTLS by default, and smartcard has a dep loop
-app-crypt/gnupg gnutls smartcard
+app-crypt/gnupg gnutls smartcard ssl
```

Comment:
Maybe drop `gnutls` here since it's not used anymore?  Also, you're enabling `smartcard`, so it should probably be dropped here.  Have you tested rebuilding the SDK and boards to see if the dependency loop is still an issue?

Reply:
Dropped the extraneous flag.

I haven't done a full sdk rebuild, but since the sdk had smartcard support before this change (just not usb+smartcard), I don't think it was relevant for the sdk already.

Reply:
Ah, okay, I missed how that flag was unmasked separately in the SDK profile.